### ISSUE TITLE
container-by-image topology can't display details

### ISSUE THREAD

**Issue by ekimekim** [weaveworks/scope#2484]
Containers by Image topology is blank unless `All Namespaces` filter is selected. And even then clicking on any node gives a 'no such node' details screen:
![screenshot](https://cloud.githubusercontent.com/assets/570052/25509381/606c864e-2b6d-11e7-9b04-86481ca0faf3.png)

Suspect this may be related to introducing the k8s filters for container views, but that's pure speculation.

**Comment by ekimekim** [comment]
These appear to be two seperate issues, as I can't reproduce the topology being blank outside prod.

**Comment by ekimekim** [comment]
I went as far back as I could in the history until I got build errors from npm modules that no longer work. It's broken on every version I try.
But it's more likely that the bug is manifesting in some way that the version changes weren't fixing it, eg. in browser local state (despite my best efforts to reset it).

**Comment by ekimekim** [comment]
After some investigation, me and @jpellizzari reckon this is a UI issue involving image names containing `/`

**Comment by ekimekim** [comment]
Found it. This problem is Weave Cloud specific. Authfe is mangling the url string by rewriting it to use a literal `/` instead of a url-escaped `%2F`. Debug logs from a local test:
```
Forwarding GET /api/topology/containers-by-image/prom%2Fmemcached-exporter%3B%3Ccontainer_image%3E to query.scope.svc.cluster.local:80, final URL: http://query.scope.svc.cluster.local:80/api/topology/containers-by-image/prom/memcached-exporter;%3Ccontainer_image%3E
```
Aligned for clarity:
```
/api/topology/containers-by-image/prom%2Fmemcached-exporter%3B%3Ccontainer_image%3E
/api/topology/containers-by-image/prom/memcached-exporter;%3Ccontainer_image%3E
```

**Comment by jpellizzari** [comment]
Closed by #1192
